consensus/tendermint/core/upon_condition_test.go

```diff
@@ -570,6 +570,128 @@ func TestOldProposal(t *testing.T) {
 		assert.Equal(t, clientLockedValue, c.lockedValue)
 		assert.Equal(t, clientLockedValue, c.validValue)
 	})
+
+	// line 28 check upon condition on prevote handler.
+	/*
+	  Please refer to the discussion history: https://github.com/clearmatics/autonity/pull/615, this test case address
+	the un-handled valid scenario which is not processed by our Tendermint implementation. The scene is depicted here
+	is a follows:
```

Comment:
This is a bit misleading, since the scenario is now handled.

```suggestion
	Please refer to the discussion history: https://github.com/clearmatics/autonity/pull/615, this test case address
	the scenario laid out below.
```